Source/CarthageKit/Archive.swift

```diff
@@ -58,12 +58,12 @@ private func untargz(archive fileURL: URL, to destinationDirectoryURL: URL) -> S
 		.then(SignalProducer<(), CarthageError>.empty)
 }
 
-private let ArchiveTemplate = "carthage-archive.XXXXXX"
+private let archiveTemplate = "carthage-archive.XXXXXX"
```

Comment:
@ikesyo What do you think of this?

The API guidelines don't specifically mention global constants, which I think should be treated differently. (Or these should move to be located on a type.)